docs/create-candy/03-validate-assets.md

```diff
@@ -5,7 +5,7 @@ sidebar_label: "2. Validate NFT Assets"
 # Validate NFT Assets
 
 :::info
-Don't skip this section. A lot of bugs happens because of invalid data.
+Don't skip this section. A lot of bugs happen because of invalid data.
```

Comment:
When used with `a lot of`, [the verb should correspond with the subject](https://english.stackexchange.com/questions/160955/is-the-sentence-a-lot-of-resources-is-available-correct), not with `a lot of`.